src/api/channels.ts

```
@@ -159,13 +159,24 @@ export async function getActiveChannels(supabase: SupabaseClient<Database>, appi
+      .order('created_at', { ascending: false }),
 
-  if (vError) {
+    supabase.from('apps')
+      .select('default_channel_ios, default_channel_android')
+      .eq('app_id', appid)
+      .single(),
+  ])
+
+  if (vError || appError) {
     log.error(`App ${appid} not found in database`)
+    console.log(vError, appError)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)